README.md

```diff
@@ -0,0 +1,15 @@
+# Phonebook
+Mozilla's internal phonebook app.
+
+## Installation
+* Clone, host on a local, PHP-enabled server, run.
+* Optional: copy ``config-local.php-dist`` to ``config-local.php`` and tweak as needed
```

Comment:
Also a copy+pastable `cp config-local.php-dist config-local.php` is always good ^_^